clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v2/description/manifest/KubernetesManifest.java

```diff
@@ -81,7 +81,7 @@ public void setName(String name) {
   @JsonIgnore
   public String getNamespace() {
     String namespace = (String) getMetadata().get("namespace");
-    return StringUtils.isEmpty(namespace) ? "default" : namespace;
+    return StringUtils.isEmpty(namespace) ? "" : namespace;
```

Comment:
I think some caching logic assumes `getNamespace` won't be empty - not entirely sure though. Did anything funny happen when you ran this?

Reply:
Nope, but I only tested it on a very tiny Kubernetes cluster, so I might not have seen it. I'll try it out at work tomorrow and see how it goes.

Reply:
Nothing funny is happening with the cluster at work either, as far as I can tell. No surprises in any logs.

Reply:
SGTM, let's merge and see what happens!